doc-content/enterprise-only/getting-started-kogito/proc-kogito-microservices-deploying-on-ocp-console.adoc

```diff
@@ -0,0 +1,100 @@
+[id="proc-kogito-microservices-deploying-on-ocp-console_{context}"]
+= Deploying {KOGITO} microservices on OpenShift using the OpenShift web console
+
+After you create your {KOGITO} microservices as part of a business application, you can use the {OPENSHIFT} web console to deploy your microservices. The {KOGITO} Operator page in the OpenShift web console guides you through the deployment process. The {KOGITO} Operator is based on the https://sdk.operatorframework.io/[Operator SDK] and automates many of the deployment steps for you. For example, when you give the operator a link to the Git repository that contains your application, the operator can automatically configure the components required to build your project from source and deploy the resulting microservices.
+
+.Prerequisites
+* The application with your {KOGITO} microservices is in a Git repository that is reachable from your OpenShift environment.
+* You have access to the OpenShift web console with `cluster-admin` permissions.
+* (Quarkus only) The `pom.xml` file of your project contains the following dependency for the https://quarkus.io/guides/microprofile-health[Quarkus `smallrye-health` extension]. This extension enables the https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes[liveness and readiness probes] that are required for Quarkus-based projects on OpenShift.
++
+.SmallRye Heath dependency for Quarkus applications on OpenShift
+[source,xml]
+----
+<dependency>
+  <groupId>io.quarkus</groupId>
+  <artifactId>quarkus-smallrye-health</artifactId>
+</dependency>
+----
+
+.Procedure
+. In the OpenShift web console, go to *Operators* -> *OperatorHub* in the left menu, search for and select *Kogito*, and follow the on-screen instructions to install the latest operator version.
```

Comment:
First two steps are for install Kogito operator itself. Could you have a seperator section above `DEPLOYING KOGITO MICROSERVICES ON OPENSHIFT USING THE OPENSHIFT WEB CONSOLE` for install kogito operator rather then mix it with deploy kogito application.

Reply:
> select _Operators_... I'm not loving 'left menu'. Is there another way you could describe it? I think I would make this two steps.
> In the OpenShift web console, go to _Operators_ -> _OperatorHub_ from the menu located on the left of the screen (or whatever).
> Search for and select _Kogito_, and follow the on-screen instructions to install the latest operator version.

@emmurphy1 If we keep it as is, the user has some action to do in the first step only, he/she does not need to look for another step to perform an action. I believe we should be having an action. wdyt?

Reply:
I'm sorry. I don't follow. There are multiple steps.